robot/Cumulus/tests/browser/gift_entry/verify_mapped_fields_tmpbldr.robot

```diff
@@ -0,0 +1,98 @@
+*** Settings ***
+
+Resource        robot/Cumulus/resources/NPSP.robot
+Library         cumulusci.robotframework.PageObjects
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+...             robot/Cumulus/resources/NPSPSettingsPageObject.py
+...             robot/Cumulus/resources/AdvancedMappingPageObject.py
+...             robot/Cumulus/resources/ObjectMangerPageObject.py
+Suite Setup     Run keywords
+...             Open Test Browser
+...             Setup Test Data
+...             Enable Gift Entry
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+*** Variables ***
+${template}       AM Template
```

Comment:
Since this is a global variable, can you declare it in all caps ?

Reply:
done